src/main/java/de/tum/in/www1/artemis/service/ParticipationService.java

```diff
@@ -138,50 +138,49 @@ public ParticipationService(GitService gitService, Optional<ContinuousIntegratio
      * @return the participation connecting the given exercise and user
      */
     public StudentParticipation startExercise(Exercise exercise, Participant participant, boolean createInitialSubmission) {
```

Comment:
`@Transactional` might be good here since we change and persist the participation

Reply:
We want to avoid @Transactional annotations. They mess up performance of the app, I was actually asked by Stephan to remove existing @Transactionals from this service